plasma_framework/contracts/src/exits/payment/PaymentStandardExitable.sol

```diff
@@ -116,24 +116,23 @@ contract PaymentStandardExitable is OnlyWithValue {
         view
     {
         bytes32 leafData = keccak256(data.rlpTx);
```

Comment:
This line could be moved down as well, to avoid hashing until necessary

Reply:
done, amended the commit